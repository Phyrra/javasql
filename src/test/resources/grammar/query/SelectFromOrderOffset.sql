select *
from t1
order by field
offset 10 rows