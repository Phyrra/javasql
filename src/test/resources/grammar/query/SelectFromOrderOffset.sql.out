SELECT *
FROM [t1]
ORDER BY [field]
OFFSET 10 ROWS